src/components/SearcForm/SearchForm.tsx

```diff
@@ -1,15 +1,39 @@
 import style from './Searchform.module.scss'
 import Button from '../Button/Button'
+import { Link, useSearchParams } from 'react-router-dom'
 
-const SearchForm = () => (
-  <form className={style.search_form}>
-    <input
-      type={'text'}
-      placeholder={'What do you want to watch?'}
-      className={style.search_form_input}
-    />
-    <Button title={'search'} classname={'search_button'} />
-  </form>
-)
+const SearchForm = () => {
+  const [search, setSearch] = useSearchParams('');
+
+  let searchTitle = '';
```

Comment:
Search Form doesn't work. You set searchTitle to `''` every time this component re-renders, you know how to preserve data in React in between renders. You need to fix this.